framework/components/ecl-forms/ecl-forms-text-inputs/package.json

```diff
@@ -1,8 +1,20 @@
 {
-  "private": true,
   "name": "@ec-europa/ecl-forms-text-inputs",
```

Comment:
Either please add `"@ec-europa/ecl-forms-help-blocks"` as a dependency or maybe just remove it for the time being.